regtests/t_cli/src/test_cli.py

```diff
@@ -29,7 +29,8 @@
 
 CLI_PYTHONPATH = f'{os.path.dirname(os.path.abspath(__file__))}/../../client/python'
 ROLE_ARN = 'arn:aws:iam::123456789012:role/my-role'
-POLARIS_URL = 'http://polaris:8181/api/catalog/v1/oauth/tokens'
+POLARIS_HOST = os.getenv('POLARIS_HOST') or 'polaris'
```

Comment:
Generally speaking this is fine, although maybe we can make the env var being used more readily visible